src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -78,14 +80,14 @@ func (cf *fetcher) pollForConfig(currentCfg yamlconf.Config, stickyConfig bool)
 				log.Debugf("Merging cloud configuration")
 				cfg := ycfg.(*Config)
 
-				err := cfg.updateFrom(bytes)
+				errr := cfg.updateFrom(bytes)
```

Comment:
As we are making change to this piece of code, we can refactor L75 to follow Go's pattern: check `err != nil` first, then `bytes == nil`, then rest of the code. That also fixes shadowing.